TileStache/Sandwich.py

```diff
@@ -241,77 +234,77 @@ def draw_stack(self, coord, tiles):
 
             else:
                 raise Core.KnownUnknown("You have to provide at least some combination of src, color and mask to Sandwich Layer")
-        
+
             #
             # Do the final composition with adjustments and blend modes.
             #
-        
+
             for (name, args) in layer.get('adjustments', []):
                 adjustfunc = adjustment_names.get(name)(*args)
                 foreground = foreground.adjust(adjustfunc)
-        
+
             opacity = float(layer.get('opacity', 1.0))
             blendfunc = blend_modes.get(layer.get('mode', None), None)
-        
+
             if mask_name:
                 rendered = rendered.blend(foreground, tiles[mask_name], opacity, blendfunc)
             else:
                 rendered = rendered.blend(foreground, None, opacity, blendfunc)
-    
+
         return rendered
 
 def local_bitmap(source, config, coord, dim):
     """ Return Blit.Bitmap representation of a raw image.
     """
     address = urljoin(config.dirpath, source)
     bytes = urlopen(address).read()
-    image = Image.open(StringIO(bytes)).convert('RGBA')
-    
+    image = Image.open(BytesIO(bytes)).convert('RGBA')
```

Comment:
not related to your changes, but can you please rename this variable to `bytes_`?